pkgs/development/web/postman/default.nix

```diff
@@ -16,10 +16,15 @@ stdenv.mkDerivation rec {
 
   buildPhase = ":";   # nothing to build
 
+  icon = fetchurl {
+    url = "https://www.getpostman.com/img-rebrand/logo.png";
```

Comment:
This looks likely to break in the future.

Reply:
namely the checksum.

Reply:
Yeah I don't like it either. The icon is isame n the package that is downloaded initially but I can't find any example using an icon from the local machine. Is there any way to do that ?

Reply:
you could take it from here: https://raw.githubusercontent.com/gilbarbara/logos/b595a2b3ecc933b77aa286bc9b76645a3203920e/logos/postman.svg

Reply:
or upload it somewhere like archive.org

Reply:
Why not take it from the software tarball ? 

I get that it's useful to download an icon *if there is none in the main package*, but why can't I use the one from the package if there is ?

Reply:
You should of course take it from the source tarball if there is one. Nobody mentioned it because everyone assumed there isn't probably an icon anymore as you added fetching it from the website.

Reply:
@fpletz could you show me an example of how to do that ?
My problem is that I can't find an example doing that.
What I want is this : 
```
  src = fetchurl {
    url = "https://dl.pstmn.io/download/version/${version}/linux64";
    sha1 = "BC0C6117BEC6D1638FD18A0E2A580617669A9297";
    name = "${name}.tar.gz";
  };

  icon = doSomethingAboutTheDownloadedPackage {}; # <--- this is what I can't figure

  desktopItem = makeDesktopItem {
    name = "postman";
    exec = "postman";
    icon = "${icon}";
    comment = "API Development Environment";
    desktopName = "Postman";
    genericName = "Postman";
    categories = "Application;Development;";
  };
```

Reply:
You can also specify the icon name:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/rocksndiamonds/default.nix#L16
and install it into the right location: https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/rocksndiamonds/default.nix#L36